source/support.tex

```diff
@@ -444,18 +498,43 @@
 \end{codeblock}
 
 \pnum
+For all members declared
+\tcode{static} \tcode{constexpr}
+in the
+\tcode{numeric_limits}
+template, specializations shall define these values in such a way
+that they are usable as
+constant expressions.
```

Comment:
Well, there are member functions declared "static constexpr" (such as `infinity()`), and users can specialize numeric_limits for their own types. Since a function can be declared "constexpr", yet not be usable in a constant expression, my guess is that this essentially requires that "T" doesn't access global or volatile variables when copied. (Such as copy is required for the return value of infinity(), for example.)